src/locales/ar.json

```diff
@@ -22,6 +22,9 @@
     "launch_get_started": "ابدأ",
     "launch_screen1_header": "طريق العودة إلى الوضع الطبيعي يبدأ من هنا"
   },
+  "onboarding": {
```

Comment:
why are all these translations included in this pr? should they be in a different commit/pr?

Reply:
Removed thank!